modules/rostests/apitests/compiler/CMakeLists.txt

```diff
@@ -1,15 +1,21 @@
 
-add_subdirectory(ms)
+if (NOT (GCC AND (ARCH STREQUAL "amd64")))
```

Comment:
```suggestion
if (MSVC OR (NOT ARCH STREQUAL "amd64"))
```

Reply:
I find this more explicit : the combination of CGG and amd64 is a problem. If someone wants to use, say, ICC, this may be fine.

Reply:
This define touches Clang too...